api/types/volume/volume_list_o_k_body.go

```diff
@@ -0,0 +1,18 @@
+package volume
```

Comment:
IMO this filename is a little funny (`api/types/volume/volume_list_o_k_body.go`) -- shouldn't `_o_k_` be just `_ok_`?

Reply:
Yeah, it's horrible; I do have some follow ups to have more sensible names for some types; my intent with this PR was initially to try keep the types the same (but unfortunately had no way to influence the filename that was generated)

Reply:
-> https://github.com/moby/moby/pull/43530